src/store/user/reducer.js

```diff
@@ -0,0 +1,24 @@
+import storage from "redux-persist/lib/storage"
+import { persistReducer } from "redux-persist"
+import { USER_SAVE } from './actions'
+
+export const initialState = null
+
+const persistConfig = () => ({
+  key: 'user',
+  storage: storage,
+})
+
+const mutations = {
+  [USER_SAVE] (state, { user }) {
+    return { ...user }
```

Comment:
Make the Immutable (use Object.freeze(...)) model for User.
Do not expand user.
You must not expand immutable objects before persisting to redux state

Reply:
fixed